lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -309,6 +309,18 @@ <h4 class="message-title">
 
 
   </section>
+  % if course_requirements:
+  <% prc_target = reverse('about_course', args=[unicode(course_requirements['courses'][0]['key'])]) %>
+  <section class="prerequisites">
+    <p class="tip">
+      ${_("You must successfully complete {link_start}{prc_display}{link_end} before you begin this course.").format(
+          link_start='<a href="{}">'.format(prc_target),
+          link_end='</a>',
+          prc_display=course_requirements['courses'][0]['display'],
```

Comment:
same nitpick comment re: explaining to code readers why we're just dereferencing the first element (for now)